client/lib/js/react/components/accounts.js

```diff
@@ -78,9 +80,27 @@ const AccountRow = React.createClass({
         activeAccount: React.PropTypes.object,
         handleClick: React.PropTypes.func
     },
-
+    
+    connectToLedger() {
+        const { account } = this.props;
+        return new BigchainDBLedger({
+            auth: {
+                account: {
+                    id: account.vk,
+                    uri: 'ws://localhost:8888/users/' + account.vk
```

Comment:
yeah, the idea is - for the future - to fetch the ledger info for an account on the rest api and use that to connect